### ISSUE TITLE
Freezes when @mentioned in a channel you're not currently in, and when a direct message appears and you aren't in that channel

### ISSUE THREAD

**Issue by Majora320** [crmarsh/discord-linux-bugs#204]
The title.
Tested under Arch Linux running the i3 window manager.
I *think* it's some sort of problem with node.js notifications under i3, because it happens with the skype client as well.

**Comment by beansmyname** [comment]
Does Discord resume when you return to the workspace on which it is running?

I'm curious because I have the same issue at launch and during the situations you've mentioned. I have a line in my i3 config that forces the client to launch on a particular workspace. 
`for_window [class="(?i)discord"] floating enable, border none, move container to workspace "3:msg"`
This is not my default workspace when first launching X. Discord will not continue to launch until I've switched to that workspace.

I suspect these may be related but not certain. If these issues are not the same, I will post a separate report.

**Comment by Majora320** [comment]
No, it doesn't, it freezes (without redrawing its window) until you manually kill the process.

**Comment by beansmyname** [comment]
Thanks. Will file a separate report later.

**Comment by XtremeHacker** [comment]
I have the same issue on the latest version on the Arch Linux AUR, anyone have ideas on why it happens?

**Comment by Majora320** [comment]
@XtremeHacker I figured it out, you need to install a notification manager like [dunst](https://dunst-project.org/) and have it run at i3 startup.

I'm not closing this issue, however, because the Discord client needs to just skip the notification phase if a notification manager isn't running.
